api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -147,13 +145,13 @@ public void sendWelcomeEmail(
         buildHtml(
             WELCOME_RESOURCE,
             welcomeMessageSubstitutionMap(
-                password, username, institutionName, showEraStepInRt, showEraStepInCt));
+                password, user.getUsername(), institutionName, showEraStepInRt, showEraStepInCt));
 
     sendWithRetries(
-        Collections.singletonList(contactEmail),
+        Collections.singletonList(user.getContactEmail()),
         Collections.emptyList(),
         "Your new All of Us Researcher Workbench Account",
-        String.format("Welcome for %s", username),
+        "Welcome for " + userForLogging(user),
```

Comment:
the main change is `userForLogging()` which now always outputs both the username and the contact email